content/dev/pmc.md

```diff
@@ -409,7 +409,9 @@ although PMCs are free to follow their own documented process for finding
 consensus on adding new committers.
 
 Once the PMC formally wants to invite an individual to be a committer, 
-it should invite the person, and require the new committer to 
+it should invite the person, 
+([use this template](https://community.apache.org/newcommitter.html#committer-invite-template))
+and require the new committer to 
```

Comment:
```suggestion
and require the new committer (if an ICLA for the committer is not already on file) to 
```

Reply:
Since this was already merged, resolved in https://github.com/apache/www-site/pull/422